lib/src/rules/control_flow_in_finally.dart

```diff
@@ -112,7 +112,7 @@ class ControlFlowInFinally extends LintRule {
 /// [ControlFlowInFinally] which is in a separate rule to allow a more granular
 /// configurability given that reporting throw statements in a finally clause is
 /// controversial.
-abstract class ControlFlowInFinallyBlockReporterMixin {
+mixin ControlFlowInFinallyBlockReporterMixin {
```

Comment:
Consider, at some point, dropping `Mixin` from the name. I don't think it adds anything.

Reply:
Works for me.  Done!